helm/auth_proxy/Chart.yaml

```
@@ -14,7 +14,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.0.1
+version: 0.0.2
```

I changed this version for testing and forgot to change it back -- my apologies!
```suggestion
version: 0.0.1
```

---

I believe you could let the version to 0.0.2 as it's present in the helm-repo, and next time we'll bump the version.

---

I think it would be confusing to keep the 0.0.2 release since nothing in the chart actually changed right?

---

It will fail because a version already exists.